src/fidesops/core/config.py

```diff
@@ -170,13 +170,20 @@ class Config:
         env_prefix = "FIDESOPS__SECURITY__"
 
 
+class UserSettings(FidesSettings):
+    """Configuration settings for Analytics variables."""
+
+    ANALYTICS_OPT_OUT: bool = True
```

Comment:
This should default to the "unset" consent state, and other code should ensure that the user is prompted for consent as soon as possible.
```suggestion
    ANALYTICS_OPT_OUT: Optional[bool]
```